dock/Dockerfile

```diff
@@ -12,8 +14,15 @@ RUN	apt-get install -y software-properties-common && \
 	apt-get install -y netcat && \
 	rm -rf /var/lib/apt/lists/*
 
+
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
+RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
+
+RUN apt-get update && \
+	apt-get install --no-install-recommends yarn
+
 #for some reason, npm install -g chokes. so we install them "locally", in the root folder.
-#RUN npm install ganache-cli truffle
+RUN npm install ganache-cli truffle
```

Comment:
we deliberately removed ganache from the "global" image, and we use ganache version installed using our `package.json`, to avoid conflicting versions.

Reply:
I have removed this command.